src/pages/iou/steps/MoneyRequstParticipantsPage/MoneyRequestParticipantsSplitSelector.js

```diff
@@ -181,6 +178,7 @@ function MoneyRequestParticipantsSplitSelector({betas, participants, personalDet
                 textInputLabel={translate('optionsSelector.nameEmailOrPhoneNumber')}
                 safeAreaPaddingBottomStyle={safeAreaPaddingBottomStyle}
                 shouldShowOptions={isOptionsDataReady}
+                shouldFocusOnSelectRow
```

Comment:
To make it consistent with mWeb, we need to import Browser and update above line
```suggestion
                shouldFocusOnSelectRow={!Browser.isMobile()}
```